pkgs/development/libraries/abseil-cpp/202301.nix

```diff
@@ -18,13 +19,17 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   cmakeFlags = [
+    "-DABSL_BUILD_TEST_HELPERS=ON"
+    "-DABSL_USE_EXTERNAL_GOOGLETEST=ON"
```

Comment:
Is this needed for https://github.com/abseil/abseil-cpp/issues/1407? if so please add a comment about it

Reply:
That is unrelated, abseil just won't look for the provided gtest if this option is not set.